Assets/Content/Systems/Player/Movement/BipedMovementController.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using Mirror;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using Vector2 = UnityEngine.Vector2;
+
+namespace SS3D.Content.Systems.Player.Movement
+{
+    /// <summary>
+    /// Controls the movement for biped characters that use the same armature
+    /// as the human model uses.
+    /// </summary>
+    [RequireComponent(typeof(BipedAnimatorController))]
+    [RequireComponent(typeof(CharacterController))]
+    [RequireComponent(typeof(Animator))]
+    public class BipedMovementController : NetworkBehaviour
```

Comment:
What's the relation of this with the `HumanoidMovementController`?

Reply:
I'll discard the HumanoidMovementController class